server/controllers/todo.js

```diff
@@ -0,0 +1,81 @@
+'use strict'
+
+const { Todo } = require('../models')
+
+class TodoController{
+// Static method for create new "todo"
+  static async createTodo(req, res, next) {
+    const inputTodo = { 
+      title: req.body.title,
+      description: req.body.description, 
+      status: req.body.status, 
+      due_date: req.body.due_date, 
+      UserId: req.userData.id
+    } 
+    try {
+      const newTodo = await Todo.create(inputTodo)
+      if (!newTodo) {
+        throw { message : msg }
```

Comment:
ini msg dari mana sam?

Reply:
dari models/todo.js nya kakk, boleh ga ya kak?

Reply:
jadi di models nya itu aku bikin constraint tambahan kak, title must not empty sama minimum due date. Nah kalo constraint nya ga fulfilled dia bakal ngethrow msg itu kak

Reply:
nanti output di postmanya jadi:
`{
    "errors": [
        "title must not empty!",
        "Minimum due date is today!"
    ]
}`